### PR TITLE
turn egauge notebook into script

### PR DESCRIPTION
**Pull request by matthew-schultz**
Took the egauge python notebook and created a python script.

Also created a test shell script that with arguments that send a request to the running egauge sensor with the id 725. 

There is also a commented out line for an offline egauge sensor to test the python script's response to error conditions (if it can't connect to the egauge sensor, etc). You may need to change the shell scripts permissions using ```chmod u+x test_egauge_api_01.sh``` from the shell while you are in the /sensors directory before running the script.

### CODE REVIEW

**Comment by matthew-schultz** [comment]
@ryantanaka helped me get rid of the merge commit, so it should be good now.

File: egauge/egauge_api.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    host = host.format(sensor_id) + '&' \
+    + minutes + '&' \
+    + output_csv + '&' \
+    + delta_compression
+    return host
+
+if __name__ == "__main__":
+    script, start, end, egauge_id, file_path = sys.argv
+
+    egauge_request(start, end, egauge_id, file_path)
```
Comment (carlosparadis) [inline egauge/egauge_api.py]: Flow my comments starting from here: Jump to line 13.

File: egauge/egauge_api.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+from io import StringIO
+
+import os
+import pandas as pd
+import requests
+import sys
+import traceback
+
+def egauge_request(start_timestamp, end_timestamp, sensor_id, output_file):
+    host = set_egauge_host(sensor_id)
```
Comment (carlosparadis) [inline egauge/egauge_api.py]: Step 2) Jump to line 52.

File: egauge/egauge_api.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+        print ('arg is ' + arg)
+
+# set host for egauge api get request
+def set_egauge_host(sensor_id, minutes='m', output_csv='c', delta_compression='C'):
+    host = 'http://egauge{}.egaug.es/cgi-bin/egauge-show?'
+    host = host.format(sensor_id) + '&' \
+    + minutes + '&' \
+    + output_csv + '&' \
+    + delta_compression
+    return host
```
Comment (carlosparadis) [inline egauge/egauge_api.py]: Step 3) Jump to line 63. Do you still remember Step 2? See comment on the PR why I had you do this exercise.

**Comment by matthew-schultz** [comment]
@carlosparadis I updated egauge_api.py to run within main(), cleaned up the code, and moved all my files to a /script subdirectory.

How does it look so far?

Thank you!

**Comment by carlosparadis** [comment]
@matthew-schultz sorry it took me so long to go over this -- it looks fine, thanks for understanding!

**Pull request merged**